mac/Keyman4MacIM/Keyman4MacIM/KeySender.m

```diff
@@ -90,23 +63,20 @@ - (void)postKeyboardEventWithSource: (CGEventSourceRef)source code:(CGKeyCode) v
 - (void)sendKeymanKeyCodeForEvent:(NSEvent *)event {
   [self.appDelegate logDebugMessage:@"KeySender sendKeymanKeyCodeForEvent"];
   
-  [self sendKeyDown:kKeymanEventKeyCode forSourceEvent:event includeKeyUp:NO];
```

Comment:
This seems like a significant change for beta?

Reply:
This did not seem to be causing an issue, but it was definitely a bug and resulted in duplicate generation of the event that triggers insertion of the text.